operator/controllers/suite_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
+Copyright 2022, 2024.
```

Comment:
Change it to
Copyright 2022, 2024 IBM Corp.

Reply:
Done